west.yml

```diff
@@ -9,16 +9,16 @@ manifest:
   remotes:
     - name: zephyrproject-rtos
       url-base: https://github.com/zephyrproject-rtos
-    - name: secomind
-      url-base: git@github.com:secomind
+    - name: astarte-platform
+      url-base: https://github.com/astarte-platform
```

Comment:
I would keep it in the SSH format

Reply:
To use ssh you must have added your public key to github, I remember that it was possible to use https authentication only via a token, not sure if they removed this possibilty. For someone who does not have their public key added this would not work

Reply:
This manifest is only used during development, we can leave the http method in the readme as the more generic method.
However, since adding the SSH key is the easiest configuration I would keep it that way for the development and ci manifests.

Reply:
I understand that the ci manifest could use the ssh format but the `west.yml` will force even who is importing edgehog using http to use ssh when fetching astarte, or am I mistaken ?